### ISSUE TITLE
No longer works

### ISSUE THREAD

**Issue by Howchie** [Artenes/ostatic#6]
Unfortunately clicking on any album or search crashes the app

**Comment by InfiniteX95** [comment]
Can confim. I have the exact same issue.
Many libraries require updates and I currently don't have the time and the knowledge to tweak the code. 
Maybe i'll try to submit a fix someday. Until then, it seems that we will still need to download albums !

**Comment by Artenes** [comment]
Yup, the website changed its HTML structure, so the code in the api package no longer works. To fix this, it is necessary to update how the content is scrapped from the KHInside website.
